x-pack/plugins/fleet/public/applications/integrations/sections/epm/screens/detail/index.tsx

```diff
@@ -209,11 +210,18 @@ export function Detail() {
             </FlexItemWithMaxHeight>
             <EuiFlexItem>
               <EuiFlexGroup alignItems="center" gutterSize="m">
-                <FlexItemWithMinWidth grow={false}>
-                  <EuiText>
-                    {/* Render space in place of package name while package info loads to prevent layout from jumping around */}
-                    <h1>{integrationInfo?.title || packageInfo?.title || '\u00A0'}</h1>
-                  </EuiText>
+                <FlexItemWithMinWidth grow={true}>
+                  <EuiFlexGroup alignItems={'center'}>
```

Comment:
nit: we tend to avoid template literals in the fleet codebase, preferring just 
```suggestion
                  <EuiFlexGroup alignItems="center">
```